app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -444,6 +450,8 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.metrics_api_port = "5432";
       $scope.emsCommonModel.default_api_port = $scope.getDefaultApiPort($scope.emsCommonModel.emstype);
       $scope.emsCommonModel.metrics_database_name = "ovirt_engine_history";
+    } else if ($scope.emsCommonModel.emstype === 'kubevirt') {
+      $scope.emsCommonModel.default_api_port = "6443";
```

Comment:
"6443"  is the correct port if kubevirt is deployed on kubernetes, iinw, for openshift is should be 8443.
since we plan on adding kubevirt as openshift child-provider, the port should be updated to set to 8443.